releases/v7.0.0.rst

```diff
@@ -63,7 +63,7 @@ Fixed
 Changed
 -------
 
-* :ref:`bulk_call` should now be done by specifying a ``Prefer: params=multiple-objects`` header. This fixes a performance regression when calling stored procedures.
+* :ref:`bulk_call` should now be done by specifying a ``Prefer: params=multiple-objects`` header. This fixes a performance regression when calling user-defined functions.
```

Comment:
Not sure about changing this older changelog.

Reply:
Hm, no. Older changelogs shouldn't be changed.